install/hst-install-debian.sh

```diff
@@ -1305,15 +1277,14 @@ if [ "$apache" = 'yes' ]; then
 	write_config_value "PROXY_PORT" "80"
 	write_config_value "PROXY_SSL_PORT" "443"
 	write_config_value "STATS_SYSTEM" "awstats"
+	write_config_value "WEB_BACKEND" "php-fpm"
 fi
 if [ "$apache" = 'no' ]; then
 	write_config_value "WEB_SYSTEM" "nginx"
 	write_config_value "WEB_PORT" "80"
 	write_config_value "WEB_SSL_PORT" "443"
 	write_config_value "WEB_SSL" "openssl"
 	write_config_value "STATS_SYSTEM" "awstats"
-fi
-if [ "$phpfpm" = 'yes' ]; then
 	write_config_value "WEB_BACKEND" "php-fpm"
```

Comment:
Duplicate of 1280

Reply:
1280 is the definition for apache and this one is for nginx. The reason it's set for both is that the end result looks cleaner like with the `write_config_value "STATS_SYSTEM" "awstats"` line.